charts/celeborn/README.md

```diff
@@ -40,17 +41,57 @@ When you want to test the template rendering, but not actually install anything.
 There are two ways to render templates. It will return the rendered template to you so you can see the output.
 
 - Local rendering chart templates
+ 
 ```shell
 helm template --debug ../celeborn
 ```
+
 - Server side rendering chart templates
+
 ```shell
 helm install --dry-run --debug --generate-name ../celeborn
 ```
-More details in [Helm Install](https://helm.sh/docs/helm/helm_install/)
-The chart can be customized using the following [celeborn configurations](https://celeborn.apache.org/docs/latest/configuration/#important-configurations)
-Specify parameters using `--set key=value[,key=value]` argument to `helm install`
+
+More details in [Helm Install](https://helm.sh/docs/helm/helm_install/).
+The chart can be customized using the following [celeborn configurations](https://celeborn.apache.org/docs/latest/configuration/#important-configurations).
+Specify parameters using `--set key=value[,key=value]` argument to `helm install`.
 
 ## Documentation
 
-For additional details on deploying the Celeborn Kubernetes Helm chart, please refer to the [Celeborn on Kubernetes](https://celeborn.apache.org/docs/latest/deploy_on_k8s/) documentation
+For additional details on deploying the Celeborn Kubernetes Helm chart, please refer to the [Celeborn on Kubernetes](https://celeborn.apache.org/docs/latest/deploy_on_k8s/) documentation.
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` | Pod affinity |
+| cluster.name | string | `"cluster"` |  |
+| dnsPolicy | string | `"ClusterFirst"` | Specifies the DNS policy for Celeborn pods to use |
+| fullnameOverride | string | `""` | String to override the default generated fullname |
+| hostNetwork | bool | `false` | Specifies whether to use the host's network namespace |
+| image.pullPolicy | string | `"Always"` | Image pull policy |
+| image.repository | string | `"aliyunemr/remote-shuffle-service"` | Image repository |
+| image.tag | string | `"0.1.1-6badd20"` | Image tag |
+| imagePullSecrets | list | `[]` | Image pull secrets for private image registry |
+| masterReplicas | int | `3` | Specifies the number of Celeborn master replicas to deploy, master replicas should not less than 3 |
```

Comment:
Why the key is still masterReplicas?

Reply:
> Why the key is still masterReplicas?

Sorry, I forget to run [helm-docs](https://github.com/norwoodj/helm-docs) go generate the latest documentation. And a pre-commit git hook can be configured to run this command automatically before commit.

Reply:
what if the developer does not have [helm-docs](https://github.com/norwoodj/helm-docs) installed locally?